pom.xml

```diff
@@ -65,10 +65,6 @@
     <system>GitHub</system>
     <url>https://github.com/Alfresco/alfresco-java-sdk/issues</url>
   </issueManagement>
-  <ciManagement>
```

Comment:
Please adapt the ciManagement section as intended. You can check https://github.com/Alfresco/alfresco-elasticsearch-connector/blob/master/pom.xml#L41-L44 as an example.

Also reflect this change on the main branch since we missed it during the default branch migration.

Reply:
Done